notes/notification_service.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import datetime
+import schedule
+import time
+import os
+import django
+import sys
+
+
+# Determine the full path to the notes.settings module
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "notes.settings")
+
+django.setup()
+
+from randomNote.models import Note
+from randomNote.serializers import NoteSerializer
+
+NATIVE_NOTIFY_URL = "https://exp.host/--/api/v2/push/send"
+APP_ID = 13004
+APP_TOKEN = "ExponentPushToken[KxFK2yJZNXIDFQzPrU5MCi]"
```

Comment:
we have this token hardcoded for now. In future it would be great to make an endpoint to accept an expo token from user to send specific notifications to specific users